api/app/src/command/medical/document/document-query.ts

```diff
@@ -97,20 +101,68 @@ export const updateDocQuery = async ({
   try {
     const existing = await PatientModel.findOne({
       where: {
-        id: patient.id,
-        cxId: patient.cxId,
+        id,
+        cxId,
       },
       lock: true,
       transaction,
     });
 
     if (existing) {
+      const convertSuccess = existing.data.documentQueryProgress?.convert?.successful
+        ? existing.data.documentQueryProgress?.convert?.successful + 1
+        : 1;
+
+      const convertError = existing.data.documentQueryProgress?.convert?.errors
+        ? existing.data.documentQueryProgress?.convert?.errors + 1
+        : 1;
+
+      const docQueryProgressStatus =
+        convertSuccess + convertError === existing.data.documentQueryProgress?.convert?.total
```

Comment:
Small detail, but could use `>=` instead of `===`, just in case the number don't match there's a scenario where we set it as completed. 😛